projecteuler/helpers/factors.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+def get_factors(number, list_to_filter_by=[]):
+    if not list_to_filter_by:
+        list_to_filter_by = list(range(1, number+1))
+    
+    factors = []
+    check_list = list_to_filter_by
```

Comment:
what is the purpose of this variable if it's always the same as list_to_filter_by?

Reply:
I think I wasn't clear if I needed to make a copy of the list for the inner loop for any reason, but their is no inner loop anymore.